openslides/urls.py

```diff
@@ -1,23 +1,27 @@
-from django.conf import settings
 from django.conf.urls import include, patterns, url
 
-from openslides.core.views import ErrorView
-from openslides.utils.plugins import get_urlpatterns
+from openslides.core.views import AngularView, ErrorView
 from openslides.utils.rest_api import router
 
+
+urlpatterns = patterns(
+    '',
+    url(r'^rest/', include(router.urls)),
+    # TODO: add "special" urls, for example pdf views etc.
+
+    url(r'^user.*', AngularView.as_view()),
+)
+
+
+# Depricated. Remove each app from the patterns below, when they should be served
+# via angular.
+
 handler403 = ErrorView.as_view(status_code=403)
```

Comment:
These error views are not deprecated I think. At least the HTTP 500 view might still appear. Maybe we have to change the template.